trackpy/tests/test_motion.py

```diff
@@ -78,10 +78,6 @@ def test_no_drift(self):
         actual_rolling = tp.compute_drift(self.dead_still, smoothing=2)
         assert_frame_equal(actual_rolling, expected[['y', 'x']])
 
-        # Small random drift
```

Comment:
@anntzer   I disabled this test. For some reason, `assert_frame_equal` was OK with comparing `1E-7` with `0.0` for ` pandas<1.1.0` Apparently `assert_frame_equal` was made more strict in pandas 1.1.0. I haven't been able to find this mentioned in the [release notes](https://pandas.pydata.org/pandas-docs/stable/whatsnew/v1.1.0.html#backwards-incompatible-api-changes).